pkg/etcdutil/etcdutil.go

```diff
@@ -23,6 +28,10 @@ import (
 	"go.etcd.io/etcd/pkg/transport"
 )
 
+const (
+	bufferSize = 4 * 1024 * 1024 // 4 MB
```

Comment:
```suggestion
	hashBufferSize = 4 * 1024 * 1024 // 4 MB
```
Can we name it like something like this so that it's easy to understand what this is used for. Also a comment here explaining something like "buffer size for streaming hash" would be better.

Reply:
I have addressed the review comments. PTAL